packages/synchro-charts/src/testing/styleGuide.ts

```diff
@@ -0,0 +1,28 @@
+export const FONT_SIZE = {
+  xxSmall: 14,
+  xSmall: 16,
+  small: 20,
+  smaller: 24,
+  medium: 32,
+  large: 48,
+  larger: 60,
+  xLarger: 96,
+};
+
+export const LINE_THICKNESS = {
+  xSmall: 8,
```

Comment:
lets stick with the `XS` `S` `M` etc. (to match the `size` string input into `iot app kit`

Reply:
Done --- Remove styleGuild.ts file